velox/dwio/parquet/writer/Writer.h

```diff
@@ -75,7 +107,8 @@ class Writer : public dwio::common::Writer {
   void close() override;
 
  private:
-  const int32_t rowsInRowGroup_;
+  uint64_t stagingRows_ = 0;
```

Comment:
We are almost done and one more small thing: these 2 counters better belong to `ArrowContext` as we store the staging data there.

Reply:
thank you for the reminder, addressed